test/hack/resource/clean/main.go

```diff
@@ -26,13 +26,18 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/iam"
 	"github.com/samber/lo"
 	"go.uber.org/zap"
+	"golang.org/x/exp/slices"
 
 	"github.com/aws/karpenter-provider-aws/test/hack/resource/pkg/metrics"
 	"github.com/aws/karpenter-provider-aws/test/hack/resource/pkg/resourcetypes"
 )
 
 const sweeperCleanedResourcesTableName = "sweeperCleanedResources"
 
+var excludedClusters = []string{
+	"soak-periodic-46287782",
```

Comment:
Could this be a tag rather than doing this by name?

Reply:
We'd have to go tag every resource individually which is a lot of work and a lot to remember. After discussion offline, I agree that checking the cluster name and the tag that we are targeting is the best way to handle this